tests/testapp/blocks.py

```diff
@@ -160,15 +166,17 @@ def simple_string(self) -> str:
 
     def simple_string_method(
         self,
-        values: Dict[str, Any] = None,
+        info: GraphQLResolveInfo,
+        values: dict[str, Any] = None,
     ):
         # Should not be used as we define `source="get_simple_string_method"`.
         raise Exception
 
     def get_simple_string_method(
         self,
-        values: Dict[str, Any] = None,
-    ) -> Optional[str]:
+        info: GraphQLResolveInfo,
+        values: dict[str, Any] = None,
+    ) -> str | None:
```

Comment:
**Question:** Why `str | None` instead of `Optional[str]`?

Reply:
this is the Python 3.10+ notation (https://docs.python.org/3/library/typing.html#typing.Optional)

Reply:
Ja, though is there a reason to choose a union over Optional? I find the union harder to read than `Optional`, though that's just a preference.

Reply:
less stuff to type. not fussed about it, really. will change

Reply:
ah, can't do that as we're using pyupgrade which enforces pep 604 typing rewrites - https://github.com/asottile/pyupgrade?tab=readme-ov-file#pep-604-typing-rewrites